addons/marketing_card/tests/test_campaign.py

```diff
@@ -261,6 +263,45 @@ def test_fetch_datetime(self):
             tokyo_header, datetime(2020, 5, 5, 21, 0, 0)
         )
 
+    @mock_image_render
+    @users('marketing_card_user')
+    def test_card_image_sync(self):
+        """Ensure all cards are properly updated, even the preview ones."""
```

Comment:
Note that we were asked to try to stop test inflation ... did not read the whole file, hence asking: can't we update an existing test instead ? Seems there are a lot of tests just for an active_test flag :)

Reply:
The flow to get this issue is relatively convoluted, but indeed as it's just a change for active_test I'll just give reproduction steps in the commit message.

That being said I feel like the issue is more around large UI tests and/or overlapping tests. I agree with reducing tests but I don't think sub 1-second tests are the big issue. More so people writing tours to check the color of divs.

Reply:
I agree, just that it is a guideline to keep in mind, hence I tell it a bit everywhere to that people are aware of it.